content/en/docs/instrumentation/net/manual.md

```diff
@@ -364,13 +364,13 @@ using var anotherActivity =
 
 ### Set Activity status
 
-A [status](/docs/concepts/signals/traces/#span-status) can be set on an
-activity, typically used to specify that an activity has not completed
-successfully - `ActivityStatusCode.Error`. In rare scenarios, you could override
-the `Error` status with `Ok`, but don't set `Ok` on successfully-completed
-spans.
+A [Status](/docs/concepts/signals/traces/#span-status) can be set on a
+[Span](/docs/concepts/signals/traces/#spans), typically used to specify that a
+Span has not completed successfully - `Error`. In rare scenarios, you could
+override the Error status with `Ok`, but don’t set `Ok` on
+successfully-completed spans.
```

Comment:
```suggestion
{{% docs/instrumentation/span-status-preamble %}}
```

This is very similar to the [docs/instrumentation/span-status-preamble](https://github.com/open-telemetry/opentelemetry.io/pull/3694/files#diff-c3dcb1499b734ee4ed4486be6f91f4dc7b83ae59981c8e57cdeada8b3e1f9947R1-R7) snippet. The only differences seem to be with capitalization of the statuses. Is it possible to use the shortcode here, or does .NET have something specific with capitalization that makes it differ from the other languages?